4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4120 @@
+{
```

Comment:
<p>You can maybe show some things here to build intuition: https://twiecki.io/blog/2014/01/02/visualizing-mcmc/</p><p> </p><p>But there are also good interactive widgets.</p><p> </p><p>I would also show a trace and explain how the sampler generated it, i.e. the fact that it's sequential and dependent on the last position.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
<p>This is a video, did it show up in NB review?</p>

Reply:
<p>No.</p>